### ISSUE TITLE
Audio files are not playing back on deployment

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#342]
Current behavior: when I look at an audio file on heliotrope, there is no audio player, just a thumbnail image of a document.

Expected behavior: when I look at an audio file on heliotrope, there is an audio player that plays back the audio.

**Comment by sethaj** [comment]
I wonder if there's a problem creating derivatives for .aiff files. In the log it looks like it starts the CreateDeriviativeJob,  but then it never finishes it. No errors, but it doesn't seem to complete? Generally a job will show the start and then completion, after which it calls the next job. We should test uploading .aiff files in isolation and see what happens.


```
production.log:I, [2016-08-02T23:55:25.634670 #17347]  INFO -- : [ActiveJob] Enqueued IngestFileJob (Job ID: fe53abc3-c40f-44d4-915d-1d9ec35a7651) to Resque(ingest) with arguments: gid://heliotrope/FileSet/dn39x155q, "/hydra-dev/heliotrope-testing/app/releases/20160802234357/tmp/uploads/dn/39/x1/55/32-Jia-Yee-Tang-AUDIO-Schnitzler Pantomime Polka.aif", nil, system, "original_file"

production.log:I, [2016-08-03T00:17:43.143611 #11528]  INFO -- : [ActiveJob] [IngestFileJob] [fe53abc3-c40f-44d4-915d-1d9ec35a7651] Performing IngestFileJob from Resque(ingest) with arguments: gid://heliotrope/FileSet/dn39x155q, "/hydra-dev/heliotrope-testing/app/releases/20160802234357/tmp/uploads/dn/39/x1/55/32-Jia-Yee-Tang-AUDIO-Schnitzler Pantomime Polka.aif", nil, nil, "original_file"

production.log:I, [2016-08-03T00:17:51.800716 #11528]  INFO -- : [ActiveJob] [IngestFileJob] [fe53abc3-c40f-44d4-915d-1d9ec35a7651] Enqueued CharacterizeJob (Job ID: 51c0ace4-1c8c-4cd3-b643-17d69e9e9bf3) to Resque(ingest) with arguments: gid://heliotrope/FileSet/dn39x155q, "dn39x155q/files/243c98ed-0f14-44f4-8fa4-b010a017e576"

production.log:I, [2016-08-03T00:40:26.968885 #23095]  INFO -- : [ActiveJob] [CharacterizeJob] [51c0ace4-1c8c-4cd3-b643-17d69e9e9bf3] Performing CharacterizeJob from Resque(ingest) with arguments: gid://heliotrope/FileSet/dn39x155q, "dn39x155q/files/243c98ed-0f14-44f4-8fa4-b010a017e576"

production.log:D, [2016-08-03T00:40:31.418573 #23095] DEBUG -- : [ActiveJob] [CharacterizeJob] [51c0ace4-1c8c-4cd3-b643-17d69e9e9bf3] Ran characterization on dn39x155q/files/243c98ed-0f14-44f4-8fa4-b010a017e576 (audio/x-aiff)

production.log:I, [2016-08-03T00:40:33.057640 #23095]  INFO -- : [ActiveJob] [CharacterizeJob] [51c0ace4-1c8c-4cd3-b643-17d69e9e9bf3] Enqueued CreateDerivativesJob (Job ID: 0695db87-4887-45ba-81f0-b2d3019d2d53) to Resque(ingest) with arguments: gid://heliotrope/FileSet/dn39x155q, "dn39x155q/files/243c98ed-0f14-44f4-8fa4-b010a017e576"

production.log:I, [2016-08-03T00:58:18.455734 #30828]  INFO -- : [ActiveJob] [CreateDerivativesJob] [0695db87-4887-45ba-81f0-b2d3019d2d53] Performing CreateDerivativesJob from Resque(ingest) with arguments: gid://heliotrope/FileSet/dn39x155q, "dn39x155q/files/243c98ed-0f14-44f4-8fa4-b010a017e576"

**Comment by jmcglone** [comment]
FWIW, while this one is important, right now I'm not sure it is as high a priority a other bugs because we don't know if we'll have the permissions and/or if NW is comfortable with including the audio files for Beta 1.

**Comment by jmcglone** [comment]
Blocked by #287

**Comment by sethaj** [comment]
It appears that x-aiff (AIFF, Audio Interchange File Format) is not supported by Hydra::Works. From https://github.com/projecthydra/hydra-works/blob/master/lib/hydra/works/models/concerns/file_set/mime_types.rb#L39

```
      def audio_mime_types
        # audio/x-wave is the mime type that fits 0.6.0 returns for a wav file.
        # audio/mpeg is the mime type that fits 0.6.0 returns for an mp3 file.
        ['audio/mp3', 'audio/mpeg', 'audio/wav', 'audio/x-wave', 'audio/x-wav', 'audio/ogg']
      end
```

**Comment by jmcglone** [comment]
@sethaj I wonder if we could convert the AIFF file to something that is supported by Hydra::Works prior to importing to heliotrope? Post Beta 1 launch we could file this issue with the Hydra::Works project?

**Comment by sethaj** [comment]
I think the plan is to not accept .aif/.aiff files for now and for the one(s) we have, convert it to .wav. In the future we could submit a request or a PR to Hydra::Works to include .aiff mime_types.

**Comment by sethaj** [comment]
We have a work around (use .wav) and #441 is a new ticket to submit the .aif PR to hydra-works.
